### ISSUE TITLE
Ensure links are distinguished from surrounding text in a way that does not rely on color (a): A11Y_TypeSpec_AI4W

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/typespec#8722]
### GIT Hub Tags:
#A11yTCS; #A11ySev3; #TypeSpec; #GH_TypeSpec_Web_Sept2025; #DesktopWeb; #A11yMAS; #WCAG1.4.1; #Use of Color;  #Win11; #FTP; #ChromiumEdge; #Rule:Link-in-text-block;

### Environment and OS details:
Application name: typespec.io
URL: https://typespec.io/docs/standard-library/built-in-decorators/
ChromiumEdge Version: 140.0.3485.66 (Official build) (64-bit)

### Reproduction Steps:

1. Open url https://typespec.io/docs/standard-library/built-in-decorators/ and login with valid Credentials.
2. "Build in -decorators" Page will get open.
3. Run accessibility insights for web.
4. Observe the Issue.

### Issue: 
Ensure links are distinguished from surrounding text in a way that does not rely on color (link-in-text-block - https://dequeuniversity.com/rules/axe/4.10/link-in-text-block?application=msftAI)

### Target application: 
Built-in Decorators | TypeSpec - https://typespec.io/docs/standard-library/built-in-decorators/

### Element path: 
td:nth-child(3) > a

### Snippet: 
<a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types">https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types</a>

### Related paths: 
  table:nth-child(73) > tbody > tr:nth-child(1) > td:nth-child(3)

### How to fix: 
Fix any of the following:
  The link has insufficient color contrast of 2.04:1 with the surrounding text. (Minimum contrast is 3:1, link text: #3d50f5, surrounding text: #353841)
  The link has no styling (such as underline) to distinguish it from the surrounding text

**Environment:** Microsoft Edge version 141.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

### Attachment:

<img width="1910" height="1029" alt="Image" src="https://github.com/user-attachments/assets/c248bf82-dac5-467c-9693-f3127d8e73b3" />

**Comment by ccbarragan** [comment]
This is the error associated with this issue:

<img width="1579" height="556" alt="Image" src="https://github.com/user-attachments/assets/e8d2d542-fbe5-4759-8995-4999b7b1e7db" />

Possibly an issue with how the URL is currently defined in the Markdown file:

https://github.com/microsoft/typespec/blob/679a0253fcf9f434df5b37f52bccb0b3eba94a44/website/src/content/docs/docs/standard-library/built-in-decorators.md?plain=1#L239

<img width="1302" height="96" alt="Image" src="https://github.com/user-attachments/assets/f17f5183-16d6-448a-ac74-a1bd83bdef77" />

To fix, recommend updating the Markdown to use `[link text](URL)` format so that it resolves correctly.

>Mime type this should apply to. The mime type should be a known mime type as described in [Common media types documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types) without any suffix (e.g. `+json`)

**Comment by kupatkar99** [comment]
verified the issue in below environment:
https://typespec.io/docs/standard-library/built-in-decorators/ 
issue is fixed. hence Closing the bug please find added attachment for reference.

<img width="1919" height="979" alt="Image" src="https://github.com/user-attachments/assets/2c627308-29c7-4538-b475-980387afd576" />
#Closed;
